### PR TITLE
bugfix-180: Modifying the Volunteer model

### PR DESCRIPTION
**Pull request by yoavmag**

### CODE REVIEW

File: client/migrations/0017_auto_20200406_2023.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='sex',
+            field=models.CharField(choices=[('MALE', 'זכר'), ('FEMALE', 'נקבה')], default=None, max_length=20),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='wanted_assignments',
+            field=multiselectfield.db.fields.MultiSelectField(choices=[('NIGHBORHOOD_COORDINATOR', 'רכז שכונה'), ('CITY_COORDINATOR', 'רכז עיר'), ('STAFF', 'מטה'), ('HAMAL', 'חמל'), ('PROJECT', 'פרויקט'), ('CHILD_CARE', 'משפחתון'), ('AGRICULTURE', 'חקלאות'), ('MISSIONS', 'משימות')], default=1, max_length=92),
```
Comment (astar26) [inline client/migrations/0017_auto_20200406_2023.py:44]: This is not according to the definition, the choices here represent volunteer type, which is a different field. Please change this to the choices specified in the issue.

File: client/migrations/0017_auto_20200406_2023.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='week_assignments_capacity',
+            field=models.IntegerField(default=1, validators=[django.core.validators.MinValueValidator(1), django.core.validators.MaxValueValidator(10)]),
+        ),
+        migrations.AlterField(
+            model_name='volunteer',
+            name='moving_way',
+            field=models.CharField(choices=[('BIKE', 'אופניים'), ('SCOOTER', 'קטנוע'), ('AUTOMOBILE', 'רכב'), ('PUBL', 'תחבצ'), ('FOOT', 'רגלית')], max_length=20),
```
Comment (astar26) [inline client/migrations/0017_auto_20200406_2023.py:54]: use תחבורה ציבורית and not תחבצ

File: client/migrations/0017_auto_20200406_2023.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        ),
+        migrations.CreateModel(
+            name='ParentalConsent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('parent_name', models.CharField(max_length=200)),
+                ('parent_id', models.CharField(max_length=9, validators=[django.core.validators.RegexValidator('^[0-9]{9}$')])),
```
Comment (astar26) [inline client/migrations/0017_auto_20200406_2023.py:61]: There's a validator written for ID's, with consideration to the check digit.

File: client/models.py [hunk 4]
```diff
@@ -76,7 +78,9 @@ class VolunteerSchedule(Timestampable):
 
 class Volunteer(Timestampable):
     MOVING_WAYS = (
-        ("CAR", "רכב"),
+        ("BIKE", "אופניים"),
+        ("SCOOTER", "קטנוע"),
+        ("AUTOMOBILE", "רכב"),
```
Comment (astar26) [inline client/models.py:83]: Use "מכונית"
Also "תחבורה ציבורית"

File: client/models.py [hunk 5]
```diff
@@ -148,22 +167,30 @@ def get_active_certificates(self):
     last_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, default="")
     organization = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
     age = models.IntegerField(null=True, blank=True, default=None)
+    sex = models.CharField(max_length=SHORT_FIELD_LENGTH, choices=SEXES, default=None)
     date_of_birth = models.DateField(null=True, default=None)
+    # What is volunteer_type? wanted_assignments is kind of the same...
     volunteer_type = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, choices=TYPES, default=DEFAULT_TYPE)
+    wanted_assignments = MultiSelectField(choices=TYPES, default=1)
```
Comment (astar26) [inline client/models.py:174]: Wrong choices. Did you intend to use "WANTED_ASSIGNMENTS"?

File: client/models.py [hunk 6]
```diff
@@ -148,22 +167,30 @@ def get_active_certificates(self):
...
+    week_assignments_capacity = models.IntegerField(default=1, validators=[MinValueValidator(1), MaxValueValidator(10)])
     areas = models.ManyToManyField(Area)
     languages = models.ManyToManyField(Language)
     phone_number = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
     email = models.EmailField(blank=True)
+    email_verified = models.BooleanField(default=False)
     city = models.ForeignKey(City, on_delete=models.CASCADE)
     neighborhood = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
     address = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
+    location_address_x = models.FloatField(default=0)
```
Comment (astar26) [inline client/models.py:184]: Change this to the PointField.

File: client/models.py [hunk 7]
```diff
@@ -76,7 +78,9 @@ class VolunteerSchedule(Timestampable):
 
 class Volunteer(Timestampable):
```
Comment (stavlocker) [inline client/models.py:79]: The following fields should be optional (either with blank=True, null=True or with a default value):

* areas
* languages
* available_starday
* keep_mandatory_worker_children
* guiding
* hearing_way

The following fields should have null=True (they currently have blank=True):

* notes
* organization
* neighborhood

File: requirements.txt [hunk 8]
```diff
@@ -7,6 +7,7 @@ django-bootstrap-form==3.4
 django-extensions==2.2.9
 django-mathfilters==1.0.0
 django-storages==1.9.1
+django-multiselectfield==0.1.12
```
Comment (stavlocker) [inline requirements.txt:10]: FYI, I'm opening an issue on deciding between the MultiSelectField and the ManyToMany with a Model (take wanted_assignments vs areas to see the difference)

Comment (stavlocker) [reply requirements.txt:10]: #199

**Pull request merged**